src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -340,17 +340,17 @@ void EnRu2_SpawnWaterMedallion(EnRu2* this, PlayState* play) {
 void EnRu2_CheckWaterMedallionCutscene(EnRu2* this, PlayState* play) {
     s32 pad[2];
     Player* player;
-    s16 temp;
+    s16 worldRotY;
 
     if ((gSaveContext.chamberCutsceneNum == CHAMBER_CS_WATER) && !IS_CUTSCENE_LAYER) {
         player = GET_PLAYER(play);
         this->action = ENRU2_AWAIT_BLUE_WARP;
         play->csCtx.script = gWaterMedallionCs;
         gSaveContext.cutsceneTrigger = 2;
         Item_Give(play, ITEM_MEDALLION_WATER);
-        temp = this->actor.world.rot.y + 0x8000;
-        player->actor.shape.rot.y = temp;
-        player->actor.world.rot.y = temp;
+        worldRotY = this->actor.world.rot.y + 0x8000;
+        player->actor.shape.rot.y = worldRotY;
+        player->actor.world.rot.y = worldRotY;
```

Comment:
considering that this gets assigned to both world and shape, it can just be `rotY` (or `yaw`, preferably).

Reply:
The point of the code here is to rotate both y rotation variables 180 degrees.